tests/setup.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test } from "@playwright/test";
+
+test("happy path setup", async ({ page }) => {
+  await page.goto("http://localhost:3000");
+
+  // setup coice
+  await page.getByLabel("Fresh Setup").click({ force: true }); // label is somehow covered
+  await page.getByRole("button", { name: "Continue" }).click();
+
+  // delete data ingo
```

Comment:
What is ingo? :)

Reply:
Haha. no idea, copy paste I guess, will fix

Reply:
...remebered